### DISCUSSION TITLE
Install software biostarbookhandbook

### DISCUSSION THREAD

**Discussion by ChaoWilliam** [biostars/biostar-handbook-issues#232] [General]
When I followed the instruction in the ebook to install the software, I faced a problem when I enter this code: curl http://data.biostarhandbook.com/install.sh | bash. It shows the error below. Does anyone know how to solve the problem? I also post the image of the output of doctor.py
![inatall problem](https://user-images.githubusercontent.com/112256796/187030442-adba0f2e-77d8-4b62-be5a-9aea1d06e301.PNG)
![doctor py output](https://user-images.githubusercontent.com/112256796/187030445-a7c3dbe9-4085-4fdb-8a5c-16b204ba2504.PNG)
.

**Comment by ialbert** [comment]
for some unexpected reason, the C compiler `gcc` did not get installed.
Perhaps a previous step was missed where you set up Linux:

https://www.biostarhandbook.com/setup-linux.html#setup-linux

do you remember whether you followed that?
